packages/lesswrong/components/tagging/ConceptItem.tsx

```diff
@@ -163,16 +165,19 @@ const ConceptItem = ({
   const titleItem = (
     <div className={classes.titleItem}>
       <div className={classes.titleItemTitle}>
-        <TagsTooltip
-          tagSlug={wikitag.slug}
-          noPrefetch
-          previewPostCount={0}
-          placement='right-start'
-        >
-          <Link to={tagGetUrl({ slug: wikitag.slug })}>
-            {wikitag.name}
-          </Link>
-        </TagsTooltip>
+        {noLinkOrHoverOnTitle
+          ? wikitag.name
```

Comment:
I assume the styling is the same in both cases?

Reply:
Yes, and appears correct